shardus_net/src/header/header_v1.rs

```diff
@@ -72,7 +75,10 @@ impl HeaderV1 {
         let mut sender_id_len_bytes = [0u8; 4];
         cursor.read_exact(&mut sender_id_len_bytes).ok()?;
         let sender_id_len = u32::from_le_bytes(sender_id_len_bytes);
-
+        if sender_id_len > BUFFER_SIZE_LIMIT_IN_BYTES {
+            error!("Sender id size exceeds the limit of {} bytes", BUFFER_SIZE_LIMIT_IN_BYTES);
+            return None;
```

Comment:
Are we also returning None in other places when using similar methods when the condition is not met?